pubs/config/spec.py

```diff
@@ -31,6 +31,25 @@
 # the full python stack is printed.
 debug = boolean(default=False)
 
+# If true the citekey is normalized using the 'citekey_format' on adding new publications.
+normalize_citekey = boolean(default=False)
+
+# String specifying how to format the citekey. All strings of 
+# the form '{{substitution:modifier}}' and '{{substitution}}' will 
+# be substituted with their appropriate values. The following
+# substitutions are used:
+#    author_last_name: last name of the first author
```

Comment:
Because it can also be the name of the editor, I think `last_name` is better. Also, the editor possibility needs to be documented.